scripts/generate-docs.sh

```diff
@@ -21,16 +21,27 @@
 # Exit on error.
 set -e
 
+PKG_NAME=$1
+URL_PREFIX=$2
+MODE=$3
```

Comment:
i think you should default `MODE` to `modules`, like you default the tsconfig file